webapp/src/modules/transaction/sagas.js

```diff
@@ -66,3 +69,7 @@ function* handleWatchLoadingTransactions(action) {
     }
   }
 }
+
+function* handleClearRequest(action) {
+  yield put(clearTransactionsSuccess(action))
```

Comment:
Maybe we can just dipatch `getTransactionHistoryFrom(state.data, action.address).map(tx => tx.hash)`, also using the selector `getTransactionHistory(state, address)` here?
cc: @cazala [reducer#83](https://github.com/decentraland/marketplace/pull/269/files#diff-778cbfb33cb84b328e53da374364a2e8R74)

Reply:
+1 to sending only `address` to `clearTransactionsRequest`